scripts/verdaccio.sh

```diff
@@ -48,6 +50,13 @@ grep -q 'http address' <(tail -f $output)
 
 # Login as test user
 yarn npm-cli-login -u abc -p abc -e 'abc@abc.com' -r $registry
```

Comment:
could probably get rid of this line for something like, didn't test it, can do later
```
(echo "abc"; sleep 2; echo "abc") | yarn npm addUser
```